src/analytics/private/sqls/redshift/sp-scan-metadata.sql

```diff
@@ -1,5 +1,6 @@
-CREATE OR REPLACE PROCEDURE {{schema}}.sp_scan_metadata(top_frequent_properties_limit NUMERIC, day_range NUMERIC)
-AS $$ 
+CREATE OR REPLACE PROCEDURE {{schema}}.sp_scan_metadata(top_frequent_properties_limit NUMERIC, day_range NUMERIC) 
+NONATOMIC AS 
```

Comment:
For this I'm not sure, if we can upsert the existing items when re-run the procedure?

@dengmingtong could you confirm if this change it's safe?

Reply:
confirmed by @dengmingtong